### ISSUE TITLE
Support Babel7

### ISSUE THREAD

**Issue by oychao** [babel/grunt-babel#84]
I'm using Babel 7, here is my package.json looks like.

```json
{
        "@babel/core": "^7.0.0-beta.36",
        "@babel/plugin-transform-react-jsx": "^7.0.0-beta.36",
        "@babel/preset-env": "^7.0.0-beta.36",
        "@babel/preset-stage-0": "^7.0.0-beta.36",
        "@babel/register": "^7.0.0-beta.36",
        "grunt": "~1.0.1",
        "grunt-babel": "^7.0.0",
}
```

and I get an error:

```bash
> grunt build

Loading "babel.js" tasks...ERROR
>> Error: Cannot find module 'babel-core'
```

Will grunt-babel support Babel 7, please?

**Comment by kaynz** [comment]
#82

**Comment by charlesbodman** [comment]
I too need this.

**Comment by danez** [comment]
Support for scoped babel 7 packages was released in 8.0.0-beta.0
